jpo-geojsonconverter/src/main/resources/schemas/processed-map.schema.json

```diff
@@ -70,7 +70,7 @@
             "additionalProperties": false
         },
         "LineStringGeometry": {
-            "type": "object",
+            "type": ["object", "string"],
             "properties": {
                 "type": {
```

Comment:
Make sure this validation still works in case the geometry is a WKT string. (Add a test with a WKT example to the MapJsonValidatorTest class).

Might need to remove  

```
"required": [
                "type",
                "coordinates"
            ],
```
or add logic so those are only required if type is object.

Reply:
I can confirm this works regardless of the child object requiring the type and coordinates. I can add a test case.

Reply:
I am going to actually push back on this for now, we don't have any tests to verify the current processed-map output even passes validation. The method that current validators are being tested are not applicable since those operate as services and are an active part of the regular application. We could add the schemas as test resources but then we have duplication. Alternatively, we could move the schemas from the main resources to the test since they are not used in the application to begin with.

Reply:
Point taken that the MapJsonValidatorTest class doesn't currently test validation of ProcessedMap at all.   Such tests could be added without moving or copying the schema, but as you say, since ProcessedMaps conformance with that schema isn't an integral concern of the geojson converter then it's not so important to have a unit test.